Makefile

```diff
@@ -42,7 +71,7 @@ changelog/add:
 	@echo " END $(CL_REMINDERS_COMMENT) -->" >> "$(CURR_VERSION_CL)"
 	@$(EDITOR) "$(CURR_VERSION_CL)"
 	@$(MAKE) changelog
-	@git add CHANGELOG.md "$(CURR_VERSION_CL)" && git commit -m "update changelog for v$(CURR_VERSION)" && \
+	@git add CHANGELOG.md "$(CURR_VERSION_CL)" && git commit -em "update changelog for v$(CURR_VERSION)" && \
```

Comment:
Would ``-e`` get much use here? it strikes me as the hardcoded message may be good enough for history, specially since it displays the correct version, and it would save the user one interactive step.

Reply:
Yeah the `-e` might be a good idea if we want to be able to write a short summary of the changelog update into the git commit message I guess... Was thinking that since the commit contains documentation anyway, we could leave the commit message plain and not offer the ability to edit the message. As a compromise, maybe we could direct the user to run `git commit --amend` if they want to add to the commit message? I'll add that before merging...